newIDE/app/src/MainFrame/index.js

```diff
@@ -3728,21 +3728,31 @@ const MainFrame = (props: Props) => {
       <TabsTitlebar
         hidden={tabsTitleBarAndEditorToolbarHidden}
         toggleProjectManager={toggleProjectManager}
-        renderTabs={onHoverEditorTab => (
+        renderTabs={(onEditorTabHovered, onEditorTabClosed) => (
           <DraggableEditorTabs
             hideLabels={false}
             editorTabs={state.editorTabs}
             onClickTab={(id: number) => _onChangeEditorTab(id)}
-            onCloseTab={(editorTab: EditorTab) => _onCloseEditorTab(editorTab)}
-            onCloseOtherTabs={(editorTab: EditorTab) =>
-              _onCloseOtherEditorTabs(editorTab)
-            }
-            onCloseAll={_onCloseAllEditorTabs}
+            onCloseTab={(editorTab: EditorTab) => {
+              // Call onEditorTabClosed before to ensure any tooltip is removed before the tab is closed.
+              onEditorTabClosed();
```

Comment:
Let's name this:
```suggestion
              onEditorTabClosing();
```

to show it's being done (while traditionnally, something that is "ed" is when the thing is finished and it's the very last thing you call)